snapshots/one/2024-09-30/oda_data_extract.py

```diff
@@ -0,0 +1,243 @@
+"""
```

Comment:
@Marigold Is this script too big for my computer? I have the message:
```
zsh: killed     python snapshots/one/2024-09-30/oda_data_extract.py
```

I will try with the server.

Reply:
Ah, I can't run it. Even if I run `make .venv` it asks me for different libraries

Reply:
I've added [instructions](https://github.com/owid/etl/pull/3342/files#diff-545c7b174f7f9d1c6f995ad153751b41bd17e2fe998b83eada6f0b096b08cb3bR23-R26) how to run the snapshot on staging. I've even tried running it, but it has been running for more than an hour and seems to be stuck. This is the output
```
ciphers.algorithms.ARC4 and will be removed from this module in 48.0.0.
  from cryptography.hazmat.primitives.ciphers.algorithms import AES, ARC4
INFO 2024-10-01 19:24:18,056 [read.py:read_crs:44] CRS data for 1960 not found. Downloading...
INFO: CRS data for 1960 not found. Downloading...
INFO 2024-10-01 19:24:18,056 [crs.py:download_crs:109] The full, detailed CRS is only available as a large file (>1GB). The package will now download the data, but it may take a while.
INFO: The full, detailed CRS is only available as a large file (>1GB). The package will now download the data, but it may take a while.
INFO 2024-10-01 19:28:01,100 [crs.py:cloud_crs_download:52] Full CRS data downloaded successfully.
INFO: Full CRS data downloaded successfully.
INFO [exchange_data.py: load_data:] Data not found, downloading...
INFO: Data not found, downloading...
INFO: Downloading DAC1 data. This may take a while...
INFO: Downloading dataflow version 1.2
INFO: Reading the dac1 schema translation
INFO: Fetching data from https://sdmx.oecd.org/public/rest/data/OECD.DCD.FSD,DSD_DAC1@DF_DAC1,1.2/.1010+11010..1140+1160...?format=csvfilewithlabels
INFO: Preprocessing the data
INFO: Data downloaded correctly.
INFO 2024-10-01 19:29:16,662 [read.py:read_multisystem:118] Multisystem data not found. Downloading...
INFO: Multisystem data not found. Downloading...
INFO 2024-10-01 19:29:16,662 [multisystem.py:download_multisystem:36] Downloading Multisystem data... This may take a while.
INFO: Downloading Multisystem data... This may take a while.
INFO 2024-10-01 19:29:45,603 [multisystem.py:cloud_multisystem_download:25] MultiSystem entire dataset.txt data downloaded and saved.
INFO: MultiSystem entire dataset.txt data downloaded and saved.
```

Give it a try, maybe you'll have more luck than me. Perhaps it needs some smart optimization?